src/hvmc_net/mutual_recursion.message

```diff
@@ -0,0 +1,14 @@
+Seems like you're trying to run some recursive function(s) on strict-mode:
```

Comment:
It's better than before, but I think it needs to be worded better.

First of all, in these cases we're 100% sure that this will result in an infinite loop and it is also very likely to allocate more and more memory.

If the float combinators optimization is not on, we should recommend activating it.

"Use combinators" is not great advice since a program can even be based only on supercombinators and still have an infinite deref loop.
The more correct version would be to "refactor the program to use lazy references", but this doesn't explain how to do it.
More concretely, the best advice is to write programs where we don't have a recursive reference in the head position of an application or in a duplication body.
In the case of applications, this rewrite can usually be achieved by lifting at least one of the  terms that contain one of the recursive calls in the chain (or making them into combinators which get automatically floated).
In the case of duplications (or pair decomposition for some cursed program using them to duplicate), we can usually fix this by manually duplicating the term with the recursive call or using a `use` expression.

Message 2 needs to containg clear and conscise version of this, and the documentation needs to be updated to clearly convey this information.